recipes/HelloConcur/README.md

```diff
@@ -0,0 +1,9 @@
+# HelloConcur
+
+A Concur port of the ["HTML - Hello" Elm Example](https://elm-lang.org/examples).
```

Comment:
```suggestion
A Concur port of the ["HTML - Hello" Elm Example](https://elm-lang.org/examples/hello).
```

Reply:
For consistency, this same change should be applied also to the README file in the HelloHalogenHooks folder.